EvidenceApi/V1/Gateways/NotifyGateway.cs

```diff
@@ -110,6 +133,18 @@ private static string GetTemplateIdFor(DeliveryMethod deliveryMethod, Communicat
             };
         }
 
+        private Dictionary<string, object> GetParamsFor(CommunicationReason communicationReason, EvidenceRequest evidenceRequest)
```

Comment:
I think because this method only has one `switch` condition, we could
- drop the switch statement
- rename it `GetParamsForDocumentUploaded`

Reply:
Same for `GetParamsFor(CommunicationReason communicationReason, DocumentSubmission documentSubmission, Resident resident)` - this could be `GetParamsForEvidenceRejected`

Reply:
We could leave `GetParamsFor(CommunicationReason communicationReason, EvidenceRequest evidenceRequest, Resident resident)` as it is, because the switch conditions may be added to when we need to send additional resident notifications

Reply:
👍